api/tests/test_regression/test_issue_482.py

```diff
@@ -0,0 +1,129 @@
+import json
+
+import requests
+
+from api.tests import APITestCase
+
+from ..util import load_content_as_json
+
+
+class Test482(APITestCase):
```

Comment:
I would be more in favor of an explicit name to make clear what is tested here without the need to look through the issues
```suggestion
class TestAliasesTracking(APITestCase):
```